### PR TITLE
Feat: Playlist

### PR DESCRIPTION
**Pull request by yashpatel1797**
Feature add
1. Create a playlist
2. delete playlist
3. add a video to the playlist
4. remove the video from the playlist

### CODE REVIEW

File: src/components/PageInfoCard/PageInfoCard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { usePlayList } from 'context'
+import { useNavigate, useParams } from "react-router-dom"
+import { deletePlaylist } from 'utilities'
+import styles from "./PageInfoCard.module.css"
+
+const PageInfoCard = () => {
+    const navigate = useNavigate();
+    const { playlists, playListDispatch } = usePlayList();
+    const { playlistId } = useParams();
+    const data = playlists.find((playlist) => playlist._id === playlistId)
```
Comment (ravikumar0403) [inline src/components/PageInfoCard/PageInfoCard.js:11]: if data is not used anywhere else, you can destructure videos here itself

File: src/components/SideBar/SideBar.js [hunk 2]
```diff
@@ -8,9 +8,13 @@ const SideBar = () => {
...
                 <ul className={styles.sidebar__links}>
                     {sideBar.map(({ name, iconName, path }) => (
-                        <li >
-                            <NavLink to={path} className={({ isActive }) => `${styles.sidebar__link_item} ${isActive ? styles.active__link : ""
-                                }`}>
+                        <li key={name}>
+                            <NavLink
+                                to={path}
+
+                                className={({ isActive }) => `${styles.sidebar__link_item} ${isActive ? styles.active__link : ""
```
Comment (ravikumar0403) [inline src/components/SideBar/SideBar.js:15]: You can use the active class inserted by react-router-dom in the html

**Review by ravikumar0403 — APPROVED** [review]
- Good folder structure
- Check for code formatting
- Good use of useRoutes and react portal

**Review by dusariudaykumar — APPROVED** [review]
-  UI Looks amazing 
- clean code good file folder structure
- clean code good file folder structure
- everything looks good

File: src/components/PageInfoCard/PageInfoCard.js (hunk 1 above)
Comment (yashpatel1797) [reply src/components/PageInfoCard/PageInfoCard.js:11]: i am using title of data.

File: src/components/SideBar/SideBar.js (hunk 2 above)
Comment (yashpatel1797) [reply src/components/SideBar/SideBar.js:15]: sure will do this.

**Pull request merged**